src/geocoders/here.ts

```diff
@@ -82,10 +86,97 @@ export class HERE implements IGeocoder {
   }
 }
 
+export class HEREv2 implements IGeocoder {
```

Comment:
Please add a TSDoc here. This allows users to lookup the API specification from the automatically generated documentation.